src/components/ArticleCard.jsx

```diff
@@ -1,10 +1,14 @@
 function ArticleCard({ article }) {
-  
-  if(article){
+
+    
+    if(article){
+      const createdAt = new Date (article.created_at)
   return (
     <div className="articlecard"> 
     <img className="articlelistimg" src={article.article_img_url}></img>
       <h3> {article.title} Article Card</h3>
+      <div>Votes {article.votes}</div>
```

Comment:
these should be p tags - div is not great for semantic html

Reply:
Updated